google/cloud/spanner/value.h

```diff
@@ -251,13 +318,46 @@ class Value {
     return MakeTypeProto(T{});
   }
   template <typename T>
```

Comment:
Note: the reason there are no blank lines between functions is because these are part of a single block of function overloads, all with a single comment. So I wasn't sure if if adding the blank line helped or hurt. I really could go either way here. I think I flipped a coin on it. If you'd prefer a blank (even a slight preference), I'm happy to change to that.